tests/acceptance/program/overview-test.js

```diff
@@ -102,7 +102,7 @@ test('change duration', function(assert) {
 test('leave duration at 1', function(assert) {
   server.create('program', {
     school: 1,
-    duration: null,
+    duration: 1
```

Comment:
@jrjohnson I had to edit few tests to make them pass. This particular one was a little tricky. I changed to the duration value because it displays `Click to edit` when duration is `null`. Let me know if this is alright.